enterprise/server/backends/distributed/distributed.go

```diff
@@ -1118,12 +1118,6 @@ func (c *Cache) GetMulti(ctx context.Context, resources []*rspb.ResourceName) (m
 			peerRequests[peer] = append(peerRequests[peer], perHashResources[0])
 		}
 		if len(peerRequests) == 0 {
-			stillMissing := make([]string, 0)
-			for h := range hashResources {
-				if _, ok := gotMap[h]; !ok {
-					stillMissing = append(stillMissing, h)
-				}
-			}
```

Comment:
The last time this was used is in a Debug log.
The log was removed in f7feb5cdb2d6216c8887eab62f0f4317f6515b97 according to 

```
git log -p -L:\ GetMulti\(:enterprise/server/backends/distributed/distributed.go -W
```